README.rst

```diff
@@ -85,6 +85,18 @@ Alternatively, you can start directly a neurodamus docker container where all th
 With the container, you can build your mod files and run simulations.
 See instructions in `docker/README.md <https://github.com/BlueBrain/neurodamus/blob/main/docker/README.md>`_.
 
+Building the docker container
+-----------------------------
+The docker container image is built automatically when a new tag is created in the repository, if a container with the specified version doesn't already exist.
+On manual pipeline runs, the container image is also built but not automatically pushed to docker-hub; you'll have to manually start the job in the pipeline you created if you want this to happen. Keep in mind that this is *optional*, the container image is *always* pushed to the gitlab registry!
```

Comment:
Could you please write where the CI needs to be launched from automatically? I still haven't found the CI run that build and publish the docker image. Today there was a new tag created. Did this trigger a special build in the CI that deployed the new image to DockerHub? If yes is there a link for it?

Reply:
I've added some extra clarification - feel free to let me know if more is needed!